x/likenft/keeper/msg_server_new_class_test.go

```diff
@@ -49,6 +49,7 @@ func TestNewClassNormal(t *testing.T) {
 	}
 }`)
 	burnable := true
+	maxSupply := uint64(0)
```

Comment:
let's test with a non zero value, since golang defaults to 0 if unfilled, this will not catch missing assignments